sprocs/instance_questions_next_allowed_grade.sql

```diff
@@ -0,0 +1,56 @@
+DROP FUNCTION IF EXISTS instance_questions_next_allowed_grade(bigint);
+CREATE OR REPLACE FUNCTION
+    instance_questions_next_allowed_grade (
+        IN iq_id BIGINT,
```

Comment:
We conventionally call these by their full names in argument lists.
```suggestion
        IN instance_question_id BIGINT,
```